src/mod/applications/mod_avmd/mod_avmd.c

```diff
@@ -1061,6 +1071,12 @@ static switch_status_t avmd_load_xml_configuration(switch_mutex_t *mutex) {
         avmd_globals.settings.detectors_lagged_n = 1;
     }
 
+    if (bad_idle_time) {
+        bad = 1;
+        switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_DEBUG, "AVMD config parameter 'detectors_idle_time' missing or invalid - using default\n");
```

Comment:
It might be helpful to add the invalid and the default value to the log.

Reply:
I'm following the code convention of the author since this is a part of the vanilla freeswitch. Adding a default value to the log might need to do this change to the rest of the code. Downside for that it can cause more code conflict when we pull the changes from the vanilla freeswitch code.